ext/liquid_c/tokenizer.c

```diff
@@ -81,7 +81,7 @@ void tokenizer_next(tokenizer_t *tokenizer, token_t *token)
           cursor++;
           token->rstrip = 1;
         }
-        if (cursor - tokenizer->cursor > 2 + token->rstrip) {
+        if (cursor - tokenizer->cursor > (intptr_t)(2 + token->rstrip)) {
```

Comment:
why cast to `intptr_t`? looking at your error message, I would have expected a cast to `int` or `unsigned int`